app/modules/purchase/accounting.rb

```diff
@@ -16,7 +16,7 @@ def variant_extra_cost_dollars
     end
 
     def tax_dollars
-      convert_cents_to_dollars(tax_cents)
+      convert_cents_to_dollars(gumroad_responsible_for_tax? ? gumroad_tax_cents : tax_cents)
```

Comment:
`gumroad_tax_cents` and `tax_cents` are mutually exclusive, and I think all of the taxes are now collected and remitted by Gumroad since becoming MoR on 1 Jan 2025.